vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data.h

```diff
@@ -37,13 +40,12 @@ class VectorData final : public Data {
 
   void Normalize();
 
-  int GetDimensionCount() const;
+  int GetDimensionCountForTesting() const;
```

Comment:
BTW, Something**ForTesting** means that you shouldn't use it outside the test code. I can be slow/limited/unstable/etc.
Chromium uses `PRESUBMIT.py` force this rule. I hope one day we will do the same.

https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md#tests-and-test_only-code